samples/Mvc/Mvc.Server/Startup.cs

```diff
@@ -43,6 +43,8 @@ public class Startup {
                 .AllowAuthorizationCodeFlow()
                 .AllowRefreshTokenFlow()
 
+                .AddEphemeralSigningKey()
```

Comment:
Alright, looks like you'll have to remove it manually. Please push a new commit to get rid of this duplicate call.